js/bundle.go

```diff
@@ -262,26 +263,31 @@ func (b *Bundle) Instantiate(logger logrus.FieldLogger, vuID uint64) (*BundleIns
 	}
 
 	rt := vuImpl.runtime
+	pgm := init.programs[b.Filename.String()]
```

Comment:
I'd suggest using a getter here instead of the map. For example, let's imagine there will be a case when nothing storing for the desired key, better in the case of stop execution, e.g., panic, whatever?

Reply:
in all the places where access this we either:
1. know that there will be the key - otherwise we have some major bug 
2. we do check that it is there and have something else to do

Given that I am going to be rewriting even more of this in #2563 or following PRs and this will just increase the amount of abstraction here. Especially not now.

Reply:
Fair enough, don't increase the abstraction, but this naked map lookup is giving me anxiety :sweat_smile: So at least add a few words of comment and/or a TODO here, that we are sure this index exists because of so and so, etc.